### ISSUE TITLE
Redraw all the documentation images in matplotlib 2.0

### ISSUE THREAD

**Issue by drvinceknight** [Axelrod-Python/Axelrod#987]
Our plots look a bit dated. This is a very simple PR: simply go through the documentation and obtain new versions of all the graphs under matplotlib 2.0.

**Comment by Epsiilon** [comment]
Hello i am in a data processing licence and i have find your project.
I have started the issue https://github.com/Axelrod-Python/Axelrod/issues/987
For the moment i have a result like this (see thereafter) and i have just one trouble with the placement of the title.
Does this correspond to what you want ?
![title_labels_payoff](https://cloud.githubusercontent.com/assets/11098300/25772809/fb450202-3272-11e7-8848-cebabc389aa3.png)

**Comment by drvinceknight** [comment]
That's exactly what we want thanks @Epsiilon although there's no need to add the titles, we're simply looking to reproduce the plots exactly as they are given by the code in the documentation. For example in http://axelrod.readthedocs.io/en/latest/tutorials/getting_started/visualising_results.html the plots that are there were made when we put a default title in place, however now we don't so it that plot needs updated :)

Here's what you need to do:

```python
import axelrod as axl
players = [axl.Cooperator(), axl.Defector(),
           axl.TitForTat(), axl.Grudger()]
players.append(axl.Random())
tournament = axl.Tournament(players)
results = tournament.play()
plot = axl.Plot(results)
p = plot.boxplot()
p.savefig("axelrod_plot.png")  # Correct path to where sphinx reads in the figure.
```

**Comment by Epsiilon** [comment]
Okay i will do that :)

**Comment by Epsiilon** [comment]
The circle around the segment is not in the same direction, is it normal ?
![title_labels_payoff](https://cloud.githubusercontent.com/assets/11098300/25773354/4e6356d8-327c-11e7-819b-ad5785ef1f38.png)
![title_labels_payoff](https://cloud.githubusercontent.com/assets/11098300/25773358/55e4ce6e-327c-11e7-8aec-3791f4ba0e8f.png)

**Comment by drvinceknight** [comment]
Yes, that's fine.

**Comment by Epsiilon** [comment]
Okay fine.

**Comment by Epsiilon** [comment]
There are an instruction for adjust the size of the out image ? But she is too small, the left indication do not appear

![demo_strategies_stackplot](https://cloud.githubusercontent.com/assets/11098300/25781844/3bd116ae-333f-11e7-8608-e18936f98cb3.png)

**Comment by Epsiilon** [comment]
Exept the previous picture, all the rest is done.

**Comment by drvinceknight** [comment]
Closed by #1004
